service/push_notification.py

```diff
@@ -56,36 +53,45 @@ def create_channels(self):
             "address": notification_url
         }
         service = Credentials.set_api_credentials(self)
-        calendars = Calendar.query.all()
+        calendars = []
         channels = []
+        for key in db.keys('*'):
+            calendar = db.hgetall(key)
+            calendar['key'] = key
+            calendars.append(calendar)
 
-        for calendar in calendars:
+        for index, calendar in enumerate(calendars):
             request_body['id'] = str(uuid.uuid4())
-            stop_channel(service, calendar.channel_id, calendar.resource_id)
+            if not 'channel_id' in calendar.keys():
+                calendar['channel_id'] = ''
+            if not 'resource_id' in calendar.keys():
+                calendar['resource_id'] = ''
+            stop_channel(service, calendar['channel_id'], calendar['resource_id'])
 
             try:
                 channel = service.events().watch(
-                    calendarId=calendar.calendar_id,
+                    calendarId=calendar['calendar_id'],
                     body=request_body).execute()
             except errors.HttpError as error:
                 print('An error occurred', error)
                 continue
 
-            update_entity_fields(
-                calendar, channel_id=channel['id'],
-                resource_id=channel['resourceId'])
-            calendar.save()
+            db.hmset(calendar['key'], {'channel_id': channel['id'], 'resource_id': channel['resourceId']})
             channels.append(channel)
 
         response = jsonify(channels)
         return response
 
     def send_notifications(self):
-        exact_calendar = Calendar.query.filter_by(
-                    resource_id=request.headers['X-Goog-Resource-Id']).first()
-        if exact_calendar.firebase_token:
+        exact_calendar = {}
```

Comment:
I think we can use this chance to rename this and use more informative names rather than exact_calendar.

Reply:
@caveinn thanks for the feedback.
`calendar` is already taken. Which name do you suggest?
I am thinking of `selected_calendar`. Sounds good?

Reply:
Done.